src/projects/berlin-open-data.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: layouts/project.liquid
+tags: ["Open Data"]
+id: berlin-open-data
+title: Berlin Open Data
+abstract: Offene Daten der Berliner Verwaltungen und landeseigenen Betriebe
+languages: ["Python", "PHP", "JavaScript"]
+license: GPL-2.0, AGPL-3.0, MIT (verschiedene Repositories)
+link: https://daten.berlin.de
+repository: https://github.com/search?q=org%3Aberlinonline+topic%3Aberlinopendata
```

Comment:
We now have a field specifically for this case: `repository_collection`. Simply rename the key here.